testdata/jsight_0.3/ser-to-openapi/req.japi.http_method.query_0.3/query_basic.openapi.todo

```diff
@@ -0,0 +1,89 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "paths": {
+    "/url": {
+      "get": {
+        "parameters": [
+          {
+            "in": "query",
+            "name": "prim1",
+            "style": "deepObject",
+            "explode": true,
+            "schema": {
+              "type": "integer",
+              "example": 1
+            },
+            "required": true
+          },
+          {
+            "in": "query",
+            "name": "prim2",
+            "style": "deepObject",
+            "explode": true,
+            "schema": {
+              "type": "string",
+              "example": "value"
+            },
+            "required": false
```

Comment:
We do not show properties by default, let's get rid of this here and in other places within the pull request.

Reply:
Fixed.